component-samples/demo/aio/WEB-INF/web.xml

```diff
@@ -316,10 +316,14 @@
         <role-name>api</role-name>
     </auth-constraint>
     <user-data-constraint>
-        <transport-guarantee>NONE</transport-guarantee>
+       <transport-guarantee>CONFIDENTIAL</transport-guarantee>
     </user-data-constraint>
   </security-constraint>
 
+  <login-config>
+       <auth-method>CLIENT-CERT</auth-method>
```

Comment:
Can we configure it via `service.yml` file ? like `<auth-method>$(AUTH_METHOD)</auth-method>`

Reply:
same for all services as resller/WEB-INF/web.xml